constants.js

```diff
@@ -70,6 +70,8 @@ const constants = {
     // https://docs.aws.amazon.com/AmazonS3/latest/API/mpUploadUploadPart.html
     maximumAllowedPartCount: 10000,
 
+    gcpMaximumAllowdPartCount: 1024,
```

Comment:
Typo: `gcpMaximumAllowdPartCount` => `gcpMaximumAllowedPartCount`

Reply:
will fix this.